com.francescogaglione.cosmicmoney.json

```diff
@@ -0,0 +1,57 @@
+{
+  "id": "com.francescogaglione.cosmicmoney",
+  "runtime": "org.freedesktop.Platform",
+  "runtime-version": "24.08",
+  "sdk": "org.freedesktop.Sdk",
+  "sdk-extensions": ["org.freedesktop.Sdk.Extension.rust-stable"],
+  "command": "cosmic_money",
+  "finish-args": [
```

Comment:
use the cosmic baseapp like other cosmic apps are doing.

Reply:
https://github.com/flathub/dev.edfloreshz.CosmicTweaks/blob/130c94510697a0df4fd49ced4163a4609afd5987/dev.edfloreshz.CosmicTweaks.json#L5-L6